### ISSUE TITLE
Test failure: Interop/PInvoke/Vector2_3_4/Vector2_3_4/Vector2_3_4.sh

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#35945]
failed in job: [runtime-coreclr jitstress 20200505.1](https://dev.azure.com/dnceng/public/_build/results?buildId=632801&view=ms.vss-test-web.build-test-results-tab&runId=19723198&resultId=100033&paneView=debug)

Error message
~~~
Return code: 1
Raw output file: /private/tmp/helix/working/A0920915/w/B18909F4/e/Interop/PInvoke/Reports/Interop.PInvoke/Vector2_3_4/Vector2_3_4/Vector2_3_4.output.txt
Raw output:
BEGIN EXECUTION
/tmp/helix/working/A0920915/p/corerun Vector2_3_4.dll ''
System.InvalidProgramException: Non-blittable parameter types are invalid for UnmanagedCallersOnly methods.
 at Vector2_3_4TestNative.PassThroughVector2ToCallback(Vector2 startingVector, Vector2Callback callback)
 at Vector2_3_4Test.RunVector2Tests() in /Users/runner/runners/2.166.4/work/1/s/src/coreclr/tests/src/Interop/PInvoke/Vector2_3_4/Vector2_3_4.cs:line 65
 at Vector2_3_4Test.Main() in /Users/runner/runners/2.166.4/work/1/s/src/coreclr/tests/src/Interop/PInvoke/Vector2_3_4/Vector2_3_4.cs:line 19
Expected: 100
Actual: 101
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=/tmp/helix/working/A0920915/p
> /private/tmp/helix/working/A0920915/w/B18909F4/e/Interop/PInvoke/Vector2_3_4/Vector2_3_4/Vector2_3_4.sh
Expected: True
Actual: False


Stack trace
   at Interop_PInvoke._Vector2_3_4_Vector2_3_4_Vector2_3_4_._Vector2_3_4_Vector2_3_4_Vector2_3_4_sh() in /Users/runner/runners/2.166.4/work/1/s/artifacts/tests/coreclr/OSX.x64.Checked/TestWrappers/Interop.PInvoke/Interop.PInvoke.XUnitWrapper.cs:line 2759
~~~

**Comment by BruceForstall** [comment]
@AaronRobinsonMSFT

**Comment by tannergooding** [comment]
I believe @AaronRobinsonMSFT already fixed this with #35926
